README.md

```diff
@@ -32,7 +32,18 @@ with clusters of apps. There will be several different kinds of clusters, such a
    - Local: http://localhost:####
 3. Your new tab should now show the React app.
 
+## Steps to Re-deploy the Java Servlet
+
+- On the root directory, run `mvn appengine:deploy`.
+
+## Steps to Re-deploy the React App
+
+1. In a seperate terminal from the Java Servlet, cd into client.
+2. Run `yarn build`.
```

Comment:
1. Running `yarn build` from the root works too, since the package.json in the root directory is linked to the one in `client` via yarn workspaces. https://github.com/googleinterns/play-web-step-2020/blob/master/package.json#L3
2. Can you add a note explaining what this does? (i.e. Builds the app for production to the build folder. It correctly bundles React in production mode and optimizes the build for the best performance.)